extensions/amp-list/0.1/amp-list.js

```diff
@@ -64,27 +67,54 @@ export class AmpList extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    return this.populateList_();
+    const populate = this.populateList_();
+    populate.catch(error => {
+      // Normally firstLayoutCallback() hides the placeholder, but it won't
+      // get called if this promise is rejected.
+      this.togglePlaceholder(false);
```

Comment:
Done, thanks. Filed https://github.com/ampproject/docs/issues/618 to update docs.